### PR TITLE
[wasm] Use interp-only instead of interp-llvmonly as the default execution mode.

### PR DESCRIPTION
**Pull request by vargaz**
interp-llvmonly caused mono_llvm_only to be set, causing bad behavior like
stack walks not working.

### CODE REVIEW

File: mono/mini/mini-runtime.h [hunk 1]
```diff
@@ -412,7 +412,7 @@ extern MonoEEFeatures mono_ee_features;
 //XXX this enum *MUST extend MonoAotMode as they are consumed together.
 typedef enum {
 	/* Always execute with interp, will use JIT to produce trampolines */
```
Comment (lewing) [inline mono/mini/mini-runtime.h:414]: Fix the comment?

**Comment by vargaz** [comment]
The #ifdef WASM stuff is still needed for aot mode.

**Comment by thaystg** [comment]
So we still need my PR?
https://github.com/mono/mono/pull/20156

**Comment by vargaz** [comment]
Don't know.

**Comment by thaystg** [comment]
Okay, I will test.

**Comment by vargaz** [comment]
@monojenkins build failed

**Pull request merged**